src/CSET/operators/plot.py

```diff
@@ -361,27 +364,42 @@ def _plot_and_save_vertical_line_series(
     fig = plt.figure(figsize=(8, 8), facecolor="w", edgecolor="k")
     iplt.plot(cube, coord, "o-")
     ax = plt.gca()
-    ax.invert_yaxis()
-    ax.set_yscale("log")
-
-    # Define y-ticks and labels for pressure log axis
-    y_tick_labels = [
-        "1000",
-        "850",
-        "700",
-        "500",
-        "300",
-        "200",
-        "100",
-        "50",
-        "30",
-        "20",
-        "10",
-    ]
-    y_ticks = [1000, 850, 700, 500, 300, 200, 100, 50, 30, 20, 10]
-
-    # Set y-axis limits and ticks
-    ax.set_ylim(1100, 100)
+
+    # test if series_coordinate is pressure level data
+    if series_coordinate == "pressure":
+        # Invert y-axis and set to log scale
+        ax.invert_yaxis()
+        ax.set_yscale("log")
+
+        # Define y-ticks and labels for pressure log axis
+        y_tick_labels = [
+            "1000",
+            "850",
+            "700",
+            "500",
+            "300",
+            "200",
+            "100",
+            "50",
+            "30",
+            "20",
+            "10",
+        ]
+        y_ticks = [1000, 850, 700, 500, 300, 200, 100, 50, 30, 20, 10]
+
+        # Set y-axis limits and ticks
+        ax.set_ylim(1100, 100)
+
+    # test if series_coordinate is model level data. The um data uses model_level_number
+    # and lfric uses full_levels as coordinate.
+    elif (
+        series_coordinate == "model_level_number" or series_coordinate == "full_levels"
```

Comment:
```suggestion
        series_coordinate in ("model_level_number", "full_levels", "half_levels")
```
Do we also need half_levels here?

Reply:
yes good call.